Jenkinsfile

```diff
@@ -95,6 +95,7 @@ pipeline {
                 stage ('Linux') {
                     steps {
                         withMaven(maven: 'maven-latest', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                            sh 'mvn javadoc:aggregate -B -DskipStatic=true -DskipTests=true -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
Would this be better after the `mvn clean install` step? Unless I'm misunderstanding, I think the `clean` command would erase the compiled docs